keps/sig-cli/1111-kui/README.md

```diff
@@ -1,30 +1,4 @@
----
-title: Kui Graphical Terminal Enhancements
-authors:
-  - "@starpit"
-  - "@paulcastro"
-owning-sig: sig-cli
-participating-sigs:
-  - sig-cli
-reviewers:
-  - "@pwittrock"
-  - "@soltysh"
-approvers:
-  - "@pwittrock"
-  - "@soltysh"
-editor: TBD
-creation-date: 2019-11-14
-last-updated: 2020-01-30
-status: implemented
-see-also:
-
-replaces:
-
-superseded-by:
-
----
-
-# Kubectl Graphical Plugins
+# KEP-1111 Kubectl Graphical Plugins
```

Comment:
1111 - doesn't seem to be a corresponding enhancement issue:

https://github.com/kubernetes/enhancements/pull/1111

Reply:
Ah sorry, our old KEP was marked 20191111.